components/feature/pwa/src/main/java/mozilla/components/feature/pwa/db/ManifestDatabase.kt

```diff
@@ -42,6 +43,21 @@ internal abstract class ManifestDatabase : RoomDatabase() {
             }
         }
 
+        @VisibleForTesting
+        internal val MIGRATION_2_3: Migration = object : Migration(2, 3) {
+            override fun migrate(database: SupportSQLiteDatabase) {
+                val cursor = database.query("SELECT * FROM manifests LIMIT 0,1")
+
+                if (cursor.getColumnIndex("share_targets") < 0) {
```

Comment:
Why do you need to do this? You know it's a migration for `2` to `3`, and this column wasn't present on `2`.

Will this code ever run in a non-migration scenario..?